tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "declaration": true,
     "outDir": "./lib",
     "strict": true,
-    "lib": ["es6", "es2017.object"],
+    "lib": ["es6", "es2017.object", "DOM"],
```

Comment:
What is the requirement for "DOM" addition?

Reply:
It was required to run `tsc`, because we're using `console.log`.